docker/builder/build.sh

```diff
@@ -3,7 +3,7 @@
 #ccache -s # uncomment to display CCache statistics
 mkdir -p /server/build_docker
 cd /server/build_docker
-cmake -G Ninja /server -DCMAKE_C_COMPILER=`which gcc-8` -DCMAKE_CXX_COMPILER=`which g++-8`
+cmake -G Ninja /server -DCMAKE_C_COMPILER=`which gcc-9` -DCMAKE_CXX_COMPILER=`which g++-9`
```

Comment:
Also fixed build script for people who use docker. Don't know if which way correct: use `g++-9` or `clang++-8`, but at least it compile with g++-9